src/partials/_global.scss

```diff
@@ -1,3 +1,45 @@
+@import url('https://fonts.googleapis.com/css2?family=Work+Sans:wght@400;600&display=swap');
+
 html {
     background-color: whitesmoke;
-}
\ No newline at end of file
+}
+
+body {
+font-family: 'Work Sans', sans-serif;
+}
+
+a {
+    text-decoration: none;
+}
+
+header {
+    margin-bottom: 1.5rem;
+    display: flex;
+    justify-content: space-between;
+}
+
+footer {
+    margin-bottom: 1.5rem;
+    color: grey;
+}
+
+li {
+    display: inline-block;
+    margin-right: 1rem;
+}
+
+.wrapper {
+    max-width: 1500px;
+    width: 85%;
+    margin: 0 auto;
+}
+
+.layout {
+    margin: 3rem auto;
+    padding: 0 1rem;
+
+}
+
+// .tnff_full_logo {
```

Comment:
you can just remove these comments if we're not planning on using it, it gets hairy when these comments start piling up